.github/workflows/pr.yaml

```
@@ -140,7 +140,7 @@ jobs:
     strategy:
       matrix:
         java: [ 11 ]
-        graalvm-version: ["22.2.0.java11"]
+        graalvm-version: ["22.2.java17"]
```

```suggestion
        graalvm-version: ["22.2.0.java17"]
```
See the failing CI action.

I think this might also be the case in https://github.com/quarkus-qe/quarkus-test-framework/pull/561, where the CI didn't show it and I missed it too.